storage/store.go

```diff
@@ -929,6 +929,11 @@ func (s *Store) Descriptor(nodeDesc *NodeDescriptor) (*StoreDescriptor, error) {
 // method, args & reply into a Raft Cmd struct and executes the
 // command using the fetched range.
 func (s *Store) ExecuteCmd(method string, args proto.Request, reply proto.Response) error {
+	select {
```

Comment:
This shouldn't be necessary if the stopper is being used correctly everywhere. What command is being executed when this code path is actually hit?

I don't want to start putting these kinds of checks everywhere in the code (as we were doing before the stopper was introduced).